src/chat/fetchingSelectors.js

```diff
@@ -1,9 +1,13 @@
 /* @flow strict-local */
-import type { Fetching, GlobalState, Narrow } from '../types';
+import { createSelector } from 'reselect';
+import type { Fetching, Narrow, Selector } from '../types';
 import { getFetching } from '../directSelectors';
 
 /** The value implicitly represented by a missing entry in FetchingState. */
 export const DEFAULT_FETCHING = { older: false, newer: false };
 
-export const getFetchingForNarrow = (state: GlobalState, narrow: Narrow): Fetching =>
-  getFetching(state)[JSON.stringify(narrow)] || DEFAULT_FETCHING;
+export const getFetchingForNarrow: Selector<Fetching, Narrow> = createSelector(
+  (state, narrow) => narrow,
+  (state, narrow) => getFetching(state),
+  (narrow, fetching) => fetching[JSON.stringify(narrow)] || DEFAULT_FETCHING,
+);
```

Comment:
Similar to `getLastMessageTopic`: this is doing O(1) work, nothing expensive, so better to drop `createSelector` entirely and just make it a function.